### PR TITLE
use shared eslint

### PR DESCRIPTION
**Pull request by oshi97**
Fixed only auto-fixable issues. Added the language-subtag-registry preamble and distributed third party notices.

J=SLAP-2003, SLAP-2064
TEST=build

### CODE REVIEW

**Review by cea2aj — APPROVED** [review]

**Review by cea2aj — CHANGES_REQUESTED** [review]
oh, looks like snyk is failing

File: package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
   "files": [
     "dist",
     "lib",
-    "src"
+    "src",
+    "THIRD-PARTY-NOTICES"
```
Comment (oshi97) [inline package.json:13]: Oh omg this was a nice catch. Would we need to add this to other repos? (maybe a separate item if we do?)

Comment (cea2aj) [reply package.json:13]: Ya, we should add this to everything we distribute. AFAIK, we only have it in answers-react-components and in slapshot-tools/packages/eslint-config-slapshot

**Review by cea2aj — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (oshi97) [reply package.json:13]: Do we also need it for the LICENSE file?

Comment (oshi97) [reply package.json:13]: Ah nvm LICENSE is automatically included

**Comment by oshi97** [comment]
I can't +2 since I opened the original PR but lgtm!

**Pull request merged**